### PR TITLE
[JQA] CLC-6843 Base new Project Sites on template

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/CLC-6843

This will be tested on J-QA before J-Dev.

### CODE REVIEW

File: app/models/canvas_lti/project_provision.rb [hunk 1]
```diff
@@ -20,13 +20,27 @@ def unique_sis_project_id
...
       project_account_id = Settings.canvas_proxy.projects_account_id
       term_id = Settings.canvas_proxy.projects_term_id
+      template_id = Settings.canvas_proxy.projects_template_id
       worker = Canvas::Course.new(user_id: @uid)
       response = worker.create(project_account_id, project_name, project_name, term_id, unique_sis_project_id)
       if (course_details = response[:body])
         site_id = course_details['id']
-        Canvas::CourseSettings.new(course_id: site_id).fix_default_view course_details
+        response = Canvas::CourseCopyImport.new(canvas_course_id: site_id).import_course_template(template_id)
+        logger.info("New Project site #{site_id} import template response: #{response}")
```
Comment (johncrossman) [inline app/models/canvas_lti/project_provision.rb]: @raydavis I don't think logging response object is a prob but I've seen `response.inspect` used to get more concise (and safer?) summary. If memory serves, with Google api we were getting key/secret in the logs.

**Comment by johncrossman** [comment]
Looks good, @raydavis  Waiting on you-know-who.

File: app/models/canvas_lti/project_provision.rb (hunk 1 above)
Comment (raydavis) [reply app/models/canvas_lti/project_provision.rb]: You're right that I'm not following the usual format -- I've been away from this code too long, I guess. I'll amend.

**Comment by raydavis** [comment]
Looks like our shared codebase / Travis is still having those sticky test issues. I think they'll clear up once we re-align with the SIS master and JRuby 2.

**Comment by johncrossman** [comment]
These failures are unrelated to the change. Merging.

**Pull request merged**